packages/astro/src/@types/astro.ts

```diff
@@ -2191,135 +2191,72 @@ export interface AstroUserConfig {
 		 *   loader: glob({ pattern: "**\/*.md", base: "./src/data/blog" }),
 		 *   schema: z.object({
 		 *     title: z.string(),
 		 *     description: z.string(),
 		 *     pubDate: z.coerce.date(),
 		 *     updatedDate: z.coerce.date().optional(),
-		 *   }),
+		 *   })
 		 * });
-		 *
-		 * export const collections = { blog };
-		 * ```
-		 *
-		 * You can load multiple entries from a single JSON file using the `file()` loader. In this case the data must either be an array of objects, which each contain an `id` property, or an object where each key is the ID.
-		 *
-		 * **Array syntax:**
-		 *
-		 * ```json
-		 * [
-		 *   {
-		 *     "id": "labrador-retriever",
-		 *     "breed": "Labrador Retriever",
-		 *     "size": "Large",
-		 *     "origin": "Canada",
-		 *     "lifespan": "10-12 years",
-		 *     "temperament": [
-		 *       "Friendly",
-		 *       "Active",
-		 *       "Outgoing"
-		 *     ]
-		 *   },
-		 *   {
-		 *     "id": "german-shepherd",
-		 *     "breed": "German Shepherd",
-		 *     "size": "Large",
-		 *     "origin": "Germany",
-		 *     "lifespan": "9-13 years",
-		 *     "temperament": [
-		 *       "Loyal",
-		 *       "Intelligent",
-		 *       "Confident"
-		 *     ]
-		 *   }
-		 * ]
-		 * ```
-		 *
-		 * **Object syntax:**
-		 *
-		 * ```json
-		 * {
-		 *   "labrador-retriever": {
-		 *     "breed": "Labrador Retriever",
-		 *     "size": "Large",
-		 *     "origin": "Canada",
-		 *     "lifespan": "10-12 years",
-		 *     "temperament": [
-		 *       "Friendly",
-		 *       "Active",
-		 *       "Outgoing"
-		 *     ]
-		 *   },
-		 *   "german-shepherd": {
-		 *     "breed": "German Shepherd",
-		 *     "size": "Large",
-		 *     "origin": "Germany",
-		 *     "lifespan": "9-13 years",
-		 *     "temperament": [
-		 *       "Loyal",
-		 *       "Intelligent",
-		 *       "Confident"
-		 *     ]
-		 *   }
-		 * }
-		 * ```
-		 *
-		 * The collection is then defined using the `file()` loader:
-		 *
-		 * ```ts
-		 * import { defineCollection, z } from 'astro:content';
-		 * import { file } from 'astro/loaders';
-		 *
+		 * 
 		 * const dogs = defineCollection({
-		 * 	loader: file('src/data/dogs.json'),
-		 * 	schema: z.object({
-		 * 		id: z.string(),
+		 *   // The path is relative to the project root, or an absolute path.
+		 *   loader: file("src/data/dogs.json"),
+		 *   schema: z.object({
+		 *     id: z.string(),
 		 *     breed: z.string(),
-		 * 		size: z.string(),
-		 * 		origin: z.string(),
-		 * 		lifespan: z.string(),
-		 * 		temperament: z.array(z.string()),
-		 * 	}),
+		 *     temperament: z.array(z.string()),
+		 *   }),
 		 * });
 		 *
-		 * export const collections = { dogs };
+		 * export const collections = { blog, dogs };
 		 * ```
 		 *
-		 * The collection can be queried in the same way as existing content collections:
+		 * #### Querying and rendering with the Content Layer API
+		 *
+		 * The collection can be queried in the same way as content collections:
```

Comment:
```suggestion
		 * The collection can be [queried in the same way as content collections](/en/guides/content-collections/#querying-collections):
```
Could be a good place for a link!